kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImpl.java

```diff
@@ -79,7 +79,7 @@ public boolean rerouteProcurementCardDocuments() {
 
         for (ProcurementCardDocument pcardDocument : documents) {
         
-            String pcardDocumentId = pcardDocument.getObjectId();
+        	String pcardDocumentId = pcardDocument.getDocumentNumber();
```

Comment:
Spacing seems a little off in the reroute method, but that's just a cosmetic issue.

For the record:
Question is: Why do we need to pause between requeuing completely **different** documents? The requeueDocument method isn't asynchronous and it waits for the DocumentRefreshQueue service to actually finish before returning. And even if it was, the next document to requeue is totally different than the current one and there really shouldn't be any secondary effects at all.
And on top of that,  even with forcing the Thread to sleep a random 3 senconds, if it gets interrupted, it just abandons the entire process by throwing a RuntimeException. If we decide to keep the sleep, I think the InterruptedException can be logged and ignored safely as it doesn't really affect the process.